nixos/modules/services/desktops/pipewire.nix

```diff
@@ -49,8 +45,13 @@ in {
         enable = mkEnableOption "JACK audio emulation";
       };
 
-      pulse = {
-        enable = mkEnableOption "PulseAudio emulation";
+      pulse-server = {
+        enable = mkEnableOption "Act as a PulseAudio server";
+      };
+
+      bluetooth = {
+        # TODO: add "-e bluez5" to 'exec pipewire-media-session' in pipewire.conf
+        enable = mkEnableOption "Load bluez5 module";
```

Comment:
Seems difficult to achieve with the current module. We could also ignore this for now if you want.

Reply:
Since upstream supports bluetooth I'd very much like to have the option to use it, I just have no idea how to do it.